login_windows.go

```diff
@@ -0,0 +1,29 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"os"
+)
+
+func readPassword() (string, error) {
+	reader := bufio.NewReader(os.Stdin)
+	pwd, _ := reader.ReadString('\n')
+
+	return pwd, nil
+}
```

Comment:
Why not use the same logic as the Linux version, using `term.ReadPassword(syscall.Stdin)` if `term.IsTerminal`, but simply failing if it is not, rather than opening `/dev/tty` as a fallback?

Reply:
It was failing to build but looking closer I just needed to convert `Stdin` to `int` and it works.  Thanks!

Reply:
updated